support-frontend/assets/pages/paypal-error/payPalError.jsx

```diff
@@ -7,39 +7,29 @@ import React from 'react';
 import Page from 'components/page/page';
 import Header from 'components/headers/header/header';
 import Footer from 'components/footer/footer';
-import CtaLink from 'components/ctaLink/ctaLink';
 import PageSection from 'components/pageSection/pageSection';
 import QuestionsContact from 'components/questionsContact/questionsContact';
 import { statelessInit as pageInit } from 'helpers/page/page';
 import { renderPage } from 'helpers/render';
 import { detect } from 'helpers/internationalisation/countryGroup';
-
+import { LinkButton } from '@guardian/src-button';
 
 // ----- Page Startup ----- //
 
 pageInit();
 
-
 // ----- Render ----- //
 
 const content = (
-  <Page
-    header={<Header countryGroupId={detect()} />}
-    footer={<Footer />}
-  >
+  <Page header={<Header countryGroupId={detect()} />} footer={<Footer />}>
     <div className="paypal-error">
-      <PageSection
-        modifierClass="paypal-error"
-      >
-        <h1 className="paypal-error__heading">PayPal Error!</h1>
+      <PageSection modifierClass="paypal-error">
+        <h1 className="paypal-error__heading">Something went wrong</h1>
         <p className="paypal-error__copy">
-          Sorry, there was a problem completing your PayPal payment. Please try again:
+          Sorry, there was a problem completing your PayPal payment. Please try
+          again:
         </p>
-        <CtaLink
-          text="Become a Supporter"
-          url="/"
-          accessibilityHint="Restart your journey to become a guardian supporter"
-        />
+        <LinkButton href="/">Try again</LinkButton>
```

Comment:
we can safely send them to `/contribute` here, since only contributions has to redirect to paypal

Reply:
Yeah makes sense 👍